internal/client/serverservice.go

```diff
@@ -0,0 +1,202 @@
+package client
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/coreos/go-oidc"
+	"github.com/hashicorp/go-retryablehttp"
+	"github.com/sirupsen/logrus"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+	"golang.org/x/oauth2/clientcredentials"
+	"golang.org/x/sync/semaphore"
+
+	"github.com/metal-toolbox/fleet-scheduler/internal/util"
+	serverserviceapi "go.hollow.sh/serverservice/pkg/api/v1"
+)
+
+const (
+	// server service attribute to look up the BMC IP Address in
+	bmcAttributeNamespace = "sh.hollow.bmc_info"
```

Comment:
this can be replaced, by referencing the const from https://github.com/metal-toolbox/rivets/blob/main/serverservice/consts.go#L8